### DISCUSSION TITLE
Where is @Romanitho?

### DISCUSSION THREAD

**Discussion by AndrewDemski-ad-gmail-com** [Romanitho/Winget-AutoUpdate#567] [General]
I'm a little concerned about @Romanitho's recent lack of activity.

There are several open topics, and development seems to have stalled without anyone to approve PRs.
Straight from the bubble, things I still remember: 
- cmtrace logging
- proxy awareness for network related operations (WAU context, not winget itself)
- expanded wiki with FAQs, readmes for IT Support Engineers and all other active users
- workflow to build intunewin files
- local self-signer for files (Looking at you, MS AppLocker)
- etc..

Does anyone know anything about his return after Christmas 2023?

**Comment by KnifMelti** [comment]
> Does anyone know anything about his return after Christmas 2023?

No, total silence it is. Makes me worry...

**Comment by AndrewDemski-ad-gmail-com** [comment]
If something goes wrong, at some point in the future the code will have to be forked to another repository and developed there.
I really wouldn't want to do that to Romain. It's his idea, his sweat and time spent.
For what it's worth, I added my email address to my GH profile bio. 
I stand by my offer to help with the topics I listed above.
Enjoy the weekend guys.

BR
AD

**Comment by Romanitho** [comment]
Hi guys, sorry for that long, long time without any news. I am very very busy, but still alive 😅
I'll be back very soon to add new things to the project.
Thanks.

**Comment by AndrewDemski-ad-gmail-com** [comment]
@Romanitho @KnifMelti 
Guys, the CMTrace logging has arrived (https://github.com/Romanitho/Winget-AutoUpdate/pull/597)
